Description: build against libsystemd library
Author: Dmitry Shachnev <mitya57@debian.org>
Contributor: Michael Healy <horsemanoffaith@gmail.com>
Forwarded: no
Last-Update: 10-31-2015

--- a/configure.ac
+++ b/configure.ac
@@ -601,7 +601,7 @@
             [with_systemd=$withval], [with_systemd=auto])
 
 PKG_CHECK_MODULES(SYSTEMD,
-                  [libsystemd-login libsystemd-daemon],
+                  [libsystemd libsystemd-daemon],
                   [have_systemd=yes], [have_systemd=no])
 
 if test "x$with_systemd" = "xauto" ; then
